UI/Web/src/app/shared/read-more/read-more.component.ts

```diff
@@ -35,6 +35,9 @@ export class ReadMoreComponent implements OnChanges {
   hideToggle: boolean = true;
   isCollapsed: boolean = true;
 
+  get displayText() {
```

Comment:
Why not just move this replacement down to the code that sets currentText? This avoids a getter which is expensive.

Reply:
Don't think I had a reason. I will change it